### DISCUSSION TITLE
State per entity on Multi-Entity Card

### DISCUSSION THREAD

**Discussion by rspierenburg** [kalkih/mini-graph-card#1085] [Q&A] [answered]
So I'm currently using mini-graph-card to show my PC temps for CPU, GPU, HDD and SSD.  I'm trying to figure out what the listed state is based off of, is the average of all the entities on the card? or just the top-most entity in the code?
Is it possible to list the entity state next to the item in the legend?
![Screenshot 2024-03-07 101159](https://github.com/kalkih/mini-graph-card/assets/33900372/66392b5d-0f03-4f66-9203-1266548ed422)

**Comment by rspierenburg** [comment]
I just figured it out using show_state and show_indicator.   Is there a way to make them all the same size?

**Reply by ildar170975** [reply, marked as answer]
When a card has 2 graphs and `show_state: true` for both of them - then these states' labels have same fontsize.
If number of displayed states >=3 - then states of the 2nd, 3rd, ... entities are shown with a smaller fontsize.
You can change fontsize manually by card-mod (not discussed here, ask in HA Community).

> Is it possible to list the entity state next to the item in the legend?

There is a registered FR, you may find it.
